src/parse_file.py

```diff
@@ -47,15 +47,30 @@ class AsmDataEntry:
     data: List[Union[str, bytes]] = attr.ib(factory=list)
     is_string: bool = attr.ib(default=False)
     is_readonly: bool = attr.ib(default=False)
+    is_bss: bool = attr.ib(default=False)
 
-    def size_bytes(self) -> int:
-        size = 0
+    def size_range_bytes(self) -> Tuple[int, int]:
```

Comment:
should probably avoid returning zero for the lower bound here, unless the upper bound is also zero.